Essentials/src/main/resources/messages.properties

```diff
@@ -466,12 +647,16 @@ month=month
 months=months
 moreCommandDescription=Fills the item stack in hand to specified amount, or to maximum size if none is specified.
 moreCommandUsage=/<command> [amount]
+moreCommandUsage1=/<command> [amount]
+moreCommandUsage1Description=Fills the held item to the specified amount, or its max size if none is specified
 moreThanZero=\u00a74Quantities must be greater than 0.
 motdCommandDescription=Views the Message Of The Day.
 motdCommandUsage=/<command> [chapter] [page]
 moveSpeed=\u00a76Set\u00a7c {0}\u00a76 speed to\u00a7c {1} \u00a76for \u00a7c{2}\u00a76.
 msgCommandDescription=Sends a private message to the specified player.
 msgCommandUsage=/<command> <to> <message>
+msgCommandUsage1=/<command> <to> <message>
+msgCommandUsage1Description=Sends the given the message to the specified player
```

Comment:
I think this should still note that it's a private message

Reply:
```suggestion
msgCommandUsage1Description=Privately sends the given message to the specified player
```